schema/SCHEMA_QUDT-v2.1.ttl

```diff
@@ -2436,6 +2436,10 @@ qudt:numericValue
   rdfs:label "numeric value" ;
   rdfs:range dtype:numericUnion ;
 .
+qudt:omUnit
+  a owl:ObjectProperty ;
+  rdfs:label "Links to the equivalent object in the OM ontology" ;
+.
 qudt:onlineReference
```

Comment:
Should we not distinguish normative and informative references, as is done for standards? I think I already have properties for this.

Reply:
I'm open to suggestions.